src/app/content/components/Page/contentLinkHandler.ts

```diff
@@ -39,24 +42,28 @@ const reducePageReferenceError = (reference: PageReferenceError, document: Docum
 };
 
 // tslint:disable-next-line: max-line-length
-const reduceReference = (reference: PageReferenceMap, currentPath: string, document: Document, systemQueryString: string) => {
+const reduceReference = (reference: PageReferenceMap, currentPath: string, document: Document, systemQueryParams: SystemQueryParams) => {
   const path = content.getUrl(reference.params);
+  const options = createNavigationOptions(systemQueryParams);
   const a = assertNotNull(
     document.querySelector(`[href^='${reference.match}']`),
     'references are created from hrefs');
```

Comment:
you can cast this to `HTMLAnchorElement` or pass it into the querySelector generic so that you don't have to fuss with it below